src/core_plugins/kibana/public/context/app.html

```diff
@@ -0,0 +1,105 @@
+<div class="kuiLocalNav">
+  <div class="kuiLocalNavRow">
+    <div class="kuiLocalNavRow__section">
+      <div class="kuiLocalBreadcrumbs">
+        <div class="kuiLocalBreadcrumb">
+          <a class="kuiLocalBreadcrumb__link" ng-href="{{ contextApp.state.navigation.discover.url }}">Discover</a>
+        </div>
+        <div class="kuiLocalBreadcrumb" ng-bind="contextApp.state.queryParameters.indexPattern.id"></div>
+      </div>
+    </div>
+  </div>
+  <div class="kuiLocalNavRow kuiLocalNavRow--secondary">
+    <div class="kuiLocalTabs">
+      <div class="kuiLocalTab kuiLocalTab-isSelected" ng-bind-template="Context of {{ contextApp.state.queryParameters.anchorUid }}"></div>
+    </div>
+  </div>
+</div>
+
+<div class="kuiViewContent kuiViewContentItem" role="main">
```

Comment:
Great that we now have error notifications panels - thanks for providing all these awesome UI widgets. 👍

Regarding the content of the error notification: I am additionally displaying the exception via the normal Kibana notification system.